lib/puppetserver/ca/action/prune.rb

```diff
@@ -18,6 +18,7 @@ class Prune
 Usage:
   puppetserver ca prune [--help]
   puppetserver ca prune [--config]
+  puppetserver ca prune [--config] [--serial HEX_NUMBER[,HEX_NUMBER]] [--certname NAME[,NAME]]
```

Comment:
We only want this particular action to run when --remove-entry is passed in, right?

Reply:
I thought it is just fine without the extra flag --remove-entry. But if it is better having that here, let me know and I can add that.

Reply:
I think it might be better with the flag, as it is a change in behavior.  Without the flag the mode is to "remove-duplicates".  It gets ambiguous as to what the behavior is if there isn't a specific mode change in my mind.  One might interpret the behavior as "remove duplicates that match this certname" without the flag

Reply:
It also gets more confusing with the next ticket where it removes only the expired entries, which is a different mode too.  
I think we should have three modes, `remove-duplicates` ,`remove-entries`, and `remove-expired` and the default is `remove-duplicates` if nothing is specified (as that is the behavior today).  If the other flags are specified without `remove-entries` that seems like an opportunity to error out and show the help?

Reply:
I guess they don't need to be mutually exclusive.  Technically we could specify any combination of them and it would still be valid.

Reply:
Added --remove-duplicates and --remove-entries. By default it will remove duplicate entries.